src/core/settings_core.json

```diff
@@ -2019,4 +2036,4 @@
       "Description": "Maximum string length for a miscellaneous string setting"
     }
   ]
-}
```

Comment:
Hi @samikhawaja could you tell me how you made this change in your editor? I can't tell the difference between the two lines.

Reply:
That's just a trailing newline. No functional change.

Reply:
The coding standard above doesn't say anything about the trailing new line. I can remove it if this is not ok.

Reply:
this is ok, no need to change it any more :)